src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String COLUMN_ID = "id";
+    private static final String COLUMN_NAME = "name";
+    private static final String COLUMN_COUNTRY = "country";
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufactureRequest = "INSERT INTO manufacturers(name, country) "
+                + "VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufactureStatement = connection.prepareStatement(
+                        insertManufactureRequest, Statement.RETURN_GENERATED_KEYS)) {
+            createManufactureStatement.setString(1, manufacturer.getName());
+            createManufactureStatement.setString(2, manufacturer.getCountry());
+            createManufactureStatement.executeUpdate();
+            ResultSet generatedKeys = createManufactureStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException throwables) {
```

Comment:
```suggestion
        } catch (SQLException e) {
```
you can simplify  it, everybody, now that it means an exception